pkg/config/template.go

```diff
@@ -178,12 +178,12 @@ type configuration struct {
 		AuditCheckURL      string        `mapstructure:"audit_check_url"`
 	} `mapstructure:"security"`
 	Email struct {
-		SMTPHost  string  `mapstructure:"smtp_host"`
-		Port      int     `mapstructure:"port"`
-		Username  string  `mapstructure:"username"`
-		Password  string  `mapstructure:"password"`
-		Sender    string  `mapstructure:"sender"`
-		RateLimit float64 `mapstructure:"rate_limit"`
-		Burst     int     `mapstructure:"burst"`
+		SMTPHost   string  `mapstructure:"smtp_host"`
+		Port       int     `mapstructure:"port"`
+		Username   string  `mapstructure:"username"`
+		Password   string  `mapstructure:"password"`
+		Sender     string  `mapstructure:"sender"`
+		LimitEvery float64 `mapstructure:"limit_every"`
```

Comment:
This parameter *must* include units.  Otherwise I have no idea.  (Or it could be `time.Duration`, which has units).